config/workspace-status.py

```diff
@@ -95,4 +93,5 @@ def get_version(path):
 
 
 if __name__ == "__main__":
+    print("STABLE_DUMMY A")
```

Comment:
```suggestion
```
?

Reply:
It is just a variable I always have there to debug (formerly in the main function), but it does not make a difference whether it is there or not, so let's get rid of it